portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACHeader.path

```diff
@@ -0,0 +1,25 @@
+<html>
+<head>
+<title>ACHeader</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ACNavigation</td></tr>
+</thead>
+<tbody>
+<!-- HEADER ELEMENTS  -->
+<tr>
+	<td>HEADER_TITLE</td>
```

Comment:
Hey @ctang3 , I noticed some locators are not being used in the macros:

HEADER_TITLE (same path present in ACIndividualsDashboard.path and ACHeader.path)
BREAKDOWN_TAB (present in ACIndividualsDashboard.path)

I don't know if you were planning to use and then gave up... or if they probably will be needed in another macro you were writing.

Reply:
Hey Alex, I added another macro to use `BREAKDOWN_TAB` in `ACIndividualsDashboard.macro` so that it'll be used :) 

For `HEADER_TITLE`, I added it since it would be used in the future for asserting the name of the page in the header (since I was making the paths file for the Header in AC I added things related to it). If you feel that we want to keep to the scope of just this ticket, I could remove it and I can make another ticket/PR for the AC Header.

Reply:
I ended up adding a macro method to utilize `HEADER_TITLE`